addons/binding/org.openhab.binding.hs110/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: HS110 Binding
+Bundle-SymbolicName: org.openhab.binding.hs110;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ org.openhab.binding.hs110,
+ org.openhab.binding.hs110.handler,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.hs110,
+ org.openhab.binding.hs110.handler
+Require-Bundle: com.google.gson,
```

Comment:
Maybe because import-package requires the specific packages to be mentioned instead of a complete osgi bundle, so you should mention all specific package which you use from gson.